thunder/benchmarks/targets.py

```diff
@@ -972,3 +979,60 @@ def test_lora_linear(benchmark, executor, compute_type, implementation):
     fn = executor(b.fn())
 
     benchmark_for_compute_type(compute_type, benchmark, fn, args, kwargs)
+
+
+#
+# optimizer benchmarks
+#
+
+
+@pytest.mark.parametrize(
+    "executor,",
+    [
+        thunderfx_executor,
+        torch_compile_executor,
+        torch_executor,
+    ],
+    ids=["thunderfx", "inductor", "eager"],
+)
+@parametrize_compute_type_only_inference
+@pytest.mark.parametrize(
+    "params,",
+    [(64, 64), (128, 64)],
+    ids=["64x64", "128x64"],
+)
+@pytest.mark.parametrize(
+    "config,",
+    [
+        ("single_tensor", False, False),
+        ("multi_tensor", True, False),
+        ("fused", False, True),
+    ],
+    ids=["single_tensor", "multi_tensor(foreach)", "fused"],
```

Comment:
```suggestion
    ids=["single_tensor", "multi_tensor", "fused"],
```
I'd avoid parenthesis in ids so that it's easier to filter with the `-k` option.